ncj/maya/render-arnold-linux/job.template.json

```diff
@@ -17,6 +17,17 @@
                 "description": "The unique name of the job."
             }
         },
+        "mayaRenderer": {
```

Comment:
This is really a version of maya opposed to a renderer?

Reply:
changing to mayaVersion

---
In reply to: [191115852](https://github.com/Azure/BatchLabs-data/pull/16#discussion_r191115852) [](ancestors = 191115852)